src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -378,7 +390,30 @@ project-with-fastly-gcs:
                 bucket: my-bucket
                 path: my-project/
                 period: 1800
-                    
+
+project-with-fastly-shield:
+    repo: ssh://git@github.com/elifesciences/dummy3
+    subdomain: www
+    aws:
+        fastly:
+            shield: true
+
+project-with-fastly-shield-pop:
+    repo: ssh://git@github.com/elifesciences/dummy3
+    subdomain: www
+    aws:
+        fastly:
+            shield:
+                pop: london-uk
+
+project-with-fastly-shield-aws-region:
+    repo: ssh://git@github.com/elifesciences/dummy3
+    subdomain: www
+    aws:
+        region: eu-central-1
```

Comment:
Hmm, not supported changing the region...

Reply:
Tried to find a few ways round it (including using `dummy-project-eu.yaml`), but can't get anything to work.

Reply:
There is more underneath that doesn't support multiple regions (e.g. the `vpc-id` and the various `subnet-*` value are specific to that VPC, which is in `us-east-1`) so it would be a separate feature for builder to support multiple regions for different projects or project instances. The `region` value is there because it's required for API calls to target the right URLs, but it isn't meant to be configurable at the moment.